client/components/jetpack-header/bluehost.js

```diff
@@ -0,0 +1,113 @@
+/** @format */
+
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+function JetpackBluehostLogo( { translate } ) {
+	return (
+		<svg width="662.5" viewBox="0 0 1325 170">
+			<title>
+				{ translate( 'Co-branded Jetpack and %(partnerName)s logo', {
+					args: {
+						partnerName: 'Bluehost',
+					},
+				} ) }
+			</title>
+			<g id="SVGs" fill="none" fillRule="evenodd">
+				<g id="Jetpack-+-Bluehost" transform="translate(97 10)">
+					<g id="gridicons-plus-small" transform="translate(229 34)">
```

Comment:
For the sake of composability (and maintainability), shouldn't we use the actual `<Gridicon icon="plus-small" />` (or `plus`) component here instead of hardcoding it as a shape here?